### PR TITLE
Update traefik.js

### PR DESCRIPTION
**Pull request by FDiskas**
I want to try traefik custom ports. If it will work - then we could improve this

### CODE REVIEW

**Comment by FDiskas** [comment]
So - config would look like:
```yml
traefikPorts:
  - '7788/tcp': 
    - HostPort: '3000'
  - '7788/udp':
    - HostPort: '3000'
```

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "docker:cleanup": "docker rm -f exoframe-server; docker rm -f exoframe-traefik",
     "cleanup": "yarn docker:cleanup",
     "lint": "eslint src/ test/ bin/exoframe-server.js",
-    "test": "NODE_ENV=testing jest --ci --runInBand --forceExit --coverage --silent",
+    "test": "NODE_ENV=testing jest --ci --runInBand --forceExit --coverage --silent --detectOpenHandles",
```
Comment (yamalight) [inline package.json]: Please remove this

**Review by yamalight — CHANGES_REQUESTED** [review]
A few small requests:
1. Please add tests for new feature
2. Please open issue / PR with docs on main repo

**Comment by FDiskas** [comment]
Sorry about allot of commits - have no ability to run tests locally hope you able to squash commits

**Review by yamalight — APPROVED** [review]

**Comment by yamalight** [comment]
Looks good, thanks!

**Pull request merged**